File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/src/Com/Aspose/Words/Model/DocumentEntry.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Com.Aspose.Words.Model 
{
  /// <summary>
  /// 
  /// </summary>  
  public class DocumentEntry 
  {           
        /// <summary>
        /// Initializes a new instance of the <see cref="DocumentEntry" /> class.
        /// </summary>
        /// <param name="Href">Href.</param>
        /// <param name="ImportFormatMode">ImportFormatMode.</param>
        public DocumentEntry(string Href = default(string), string ImportFormatMode = default(string))
        {
                        this.Href = Href;
                        this.ImportFormatMode = ImportFormatMode;
        }

        /// <summary>
        /// Gets or sets Href
        /// </summary>    
        public string Href { get; set; }

        /// <summary>
        /// Gets or sets ImportFormatMode
        /// </summary>    
        public string ImportFormatMode { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class DocumentEntry {\n");
          sb.Append("  Href: ").Append(Href).Append("\n");
          sb.Append("  ImportFormatMode: ").Append(ImportFormatMode).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}

File: Examples/Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/DocumentEntry.java
package com.aspose.words.model;

public class DocumentEntry {
  private String Href = null;
  private String ImportFormatMode = null;
  /**
	 * getHref
	 * Gets String
	 * @return Href
	 */
  public String getHref() {
    return Href;
  }

	/**
	 * setHref
	 * Sets String
	 * @param Href String
	 */
  public void setHref(String Href) {
    this.Href = Href;
  }

  /**
	 * getImportFormatMode
	 * Gets String
	 * @return ImportFormatMode
	 */
  public String getImportFormatMode() {
    return ImportFormatMode;
  }

	/**
	 * setImportFormatMode
	 * Sets String
	 * @param ImportFormatMode String
	 */
  public void setImportFormatMode(String ImportFormatMode) {
    this.ImportFormatMode = ImportFormatMode;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class DocumentEntry {\n");
    sb.append("  Href: ").append(Href).append("\n");
    sb.append("  ImportFormatMode: ").append(ImportFormatMode).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

